lib/rabbitmq/sender.go

```diff
@@ -0,0 +1,54 @@
+package rabbitmq
+
+import (
+	"context"
+	"encoding/json"
+	"time"
+
+	"github.com/pkg/errors"
+	amqp "github.com/rabbitmq/amqp091-go"
+)
+
+type Sender struct {
+	ch *amqp.Channel
+}
+
+func NewSender(conn *amqp.Connection) (*Sender, error) {
+	ch, err := conn.Channel()
+	if err != nil {
+		return nil, errors.Wrap(err, "Failed to open a channel")
+	}
+
+	return &Sender{
+		ch: ch,
+	}, nil
+}
+
+func (s *Sender) SendMessage(queueName string, msg any) error {
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
```

Comment:
pass context to argument, instead of create a new background context every time. It's good practice.